src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -7,47 +7,56 @@
 import java.io.FileWriter;
 
 public class WorkWithFile {
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
this builder should belong to method where it is used, no need to move it to class field